app/views/pages/rollover_recruitment.html.erb

```diff
@@ -1,19 +1,19 @@
-<%= content_for :page_title, "Recruiting for the 2021 to 2022 cycle" %>
+<%= content_for :page_title, "Recruiting for the #{next_recruitment_cycle_period_text} cycle" %>
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <h1 class="govuk-heading-xl">
-      Recruiting for the <br>2021 to 2022 cycle
+      Recruiting for the <br><%= next_recruitment_cycle_period_text %> cycle
     </h1>
 
     <p class="govuk-body">
       You do not need to request permission to recruit for any courses other than
-      fee-funded PE. Accredited bodies should request fee-funded PE by 10 July 2020.
+      fee-funded PE. Accredited bodies should request fee-funded PE by 10 July.
```

Comment:
Should we put `Settings.allocation_cycle_year` here?

Reply:
There's another card for making these dates into settings.